tests/integration-tests/test_runner.py

```diff
@@ -656,6 +673,12 @@ def _set_ami_args(args, pytest_args):
     if args.ami_owner:
         pytest_args.extend(["--ami-owner", args.ami_owner])
 
+    if args.available_amis_oss_x86:
+        pytest_args.extend(["--available-amis-oss-x86", " ".join(args.available_amis_oss_x86)])
```

Comment:
Why do we need to specify `available-amis-` in this option?
If I use an option like this I assume that the OSS I'm specifying are only those that are available.

Reply:
There is an existing `-oss` parameter in test_runner, which has a different meaning. I used a more verbose name to be distinguished

Reply:
If I understood correctly, the difference between `--oss` and `--available-amis-oss-x86` is that the list of OSs for the second one is going to be rotated.

If this is the case, what about `--rotating-oss-x86` ?